drfc_manager/config_env.py

```
@@ -1,90 +1,85 @@
+    bucket_name: str = Field(
+        default="tcc-experiments",
+        description="Default bucket for DeepRacer models/files",
+    )
+    custom_files_folder: str = Field(
+        default="custom_files",
+        description="S3 prefix for custom files (reward fn, hyperparams)",
+    )
 
-    @validator('server_url', pre=True)
+    @validator("server_url", pre=True)
     def ensure_http_scheme(cls, v):
```

Pylint (3.3.7)

[error] Method 'ensure_http_scheme' should have "self" as first argument

(E0213)